polkadot-protocol/protocol-components/transactions/transactions-lifecycle.md

```diff
@@ -0,0 +1,133 @@
+
+![](/images/polkadot-protocol/polkadot-components/transactions/transactions-lifecycles/transaction-lifecycle-1.webp)
+
+Any signed or unsigned transaction sent to a non-authoring node is gossiped to other nodes in the network and enters their transaction pool until an authoring node receives it.
+
+## Validating and Queuing Transactions
+
+As discussed in the [Consensus](TODO:update-path){target=\_blank} section, most nodes in the network must agree on the order of transactions in a block to decide on the state of the blockchain and to continue securely adding blocks. To reach a consensus, two-thirds of the nodes must agree on the order of the transactions executed and the resulting state change. To prepare for consensus, transactions are first validated and queued on the local node in a transaction pool.
+
+### Validating Transactions in the Transaction Pool
+
+Using rules defined in the runtime, the transaction pool checks the validity of each transaction. The checks ensure that only valid transactions that meet specific conditions are queued to be included in a block. For example, the transaction pool might perform the following checks to determine whether a transaction is valid:
+
+- Is the transaction index—also called the transaction nonce—correct?
+- Does the account used to sign the transaction have enough funds to pay the associated fees?
+- Is the signature used to sign the transaction valid?
+
+After the initial validity check, the transaction pool periodically checks whether existing transactions are valid. If a transaction is found to be invalid or has expired, it is dropped from the pool.
+
+The transaction pool only deals with the validity of the transaction and the ordering of valid transactions placed in a transaction queue. Specific details on how the validation mechanism works—including handling fees, accounts, or signatures—can be found in the [`validate_transaction`](https://paritytech.github.io/polkadot-sdk/master/sp_transaction_pool/runtime_api/trait.TaggedTransactionQueue.html#method.validate_transaction){target=\_blank} method.
+
+### Adding Valid Transactions to a Transaction Queue
+
+If a transaction is identified as valid, the transaction pool moves the transaction into a transaction queue. There are two transaction queues for valid transactions:
+
+- The ready queue contains transactions that can be included in a new pending block. If the runtime is built with FRAME, transactions must be placed in the exact order in which they are placed in the ready queue.
+
+- The future queue contains transactions that might become valid in the future. For example, if a transaction has a nonce that is too high for its account, it can wait in the future queue until the appropriate number of transactions for the account have been included in the chain.
```

Comment:
```suggestion
- The ready queue contains transactions that can be included in a new pending block. If the runtime is built with FRAME, transactions must be placed in the exact order in which they are placed in the ready queue

- The future queue contains transactions that might become valid in the future. For example, if a transaction has a nonce that is too high for its account, it can wait in the future queue until the appropriate number of transactions for the account have been included in the chain
```